src/main.rs

```diff
@@ -312,6 +316,7 @@ fn parse_args(raw_args: Vec<std::ffi::OsString>) -> Result<Args, pico_args::Erro
         bin:                    input.opt_value_from_str("--bin")?,
         example:                input.opt_value_from_str("--example")?,
         test:                   input.opt_value_from_str("--test")?,
+        z:                      input.opt_value_from_str("-Z")?,
```

Comment:
It has to be `input.values_from_str("-Z")`

Reply:
Ok, I'll test it.